packages/tkg-clusterclass-aws/bundle/config/upstream/overlay-pod-security-admission.yaml

```
@@ -64,71 +64,104 @@ spec:
                 items:
                   type: string
                 description: "namespaces excluded to apply PodSecurityConfiguration Admission."
-  #@overlay/match missing_ok=True
-  patches:
   #@overlay/remove
   #@overlay/match by=overlay.map_key("name"),missing_ok=True
-  - name: podSecurityStandard
+  - name: apiserverEventRateLimitConf
```

```suggestion
  - name: podSecurityStandard
```

The removal stays the same, the upstream templates have a `podSecurityStandard` entry which needs to get removed if exists.